app/src/NativeBase/Assets/Icons/index.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * @file Exports icons for easy access elsewhere.
+ */
+
+export { default as Home } from './Home.svg'
+export { default as Projects } from './Projects.svg'
+export { default as EventTicket } from './EventTicket.svg'
```

Comment:
For consistency, if we're naming icons here by what they are for (e.g. Projects) rather than what they look like visually, I think it would be more consistent to just call this `Events`

Reply:
You're quite right Joe, this should be more consistent. I've given it some thought and very much prefer icon names that describe what they look like, making it easier to use them for other relevant purposes if appropriate. Some icons of course are an exception to that rule (such as home and search), and in this case the home icon is still called home.